modules/parent-join/src/main/java/org/elasticsearch/join/mapper/Relations.java

```diff
@@ -53,4 +61,20 @@ public int hashCode() {
     public String toString() {
         return parent + "->" + children;
     }
+
+    public static List<Relations> parse(Object fieldNode) {
```

Comment:
does this need to be public?

Reply:
It does not, I'll make it package-private.